### ISSUE TITLE
Add more developer tools functionality

### ISSUE THREAD

**Issue by ColdIce1** [FlashpointProject/launcher#263]
By "Developer Tools" I mean the buttons you find at the developer tab that does things like check for missing images or if any of the playlists are broken.

Developer tools that might be useful:
- Tool: Find all URLs that start with https or "https (not case-sensitive)
- Tool: Find all games with Application Path values that does not point to a file.
- Tool: Find all games with Launch Command value that does not point to a file (after being redirected).

Suggestion from nosamu:

Basically take each URL in the XML, convert it to a path, and check if that path exists in Flashpoint Ultimate with that exact capitalization

https://trello.com/c/QsRpo5rh/28-new-developer-tools
